src/functions-and-arrays.js

```diff
@@ -78,7 +145,23 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(array) {
```

Comment:
Here you needed another parameter to pass what was searched
```suggestion
function howManyTimes(array, search) {
```